pkg/rabbit/queue_test.go

```diff
@@ -179,8 +179,9 @@ func TestNewQueue(t *testing.T) {
 		{
 			name: "creates a queue for source",
 			args: &rabbit.QueueArgs{
-				Name:      sourceName,
-				Namespace: namespace,
+				Name:          sourceName,
+				Namespace:     namespace,
+				RabbitMQVhost: "test",
```

Comment:
nit: I think a lot of the argument names can just be `Vhost` instead of `RabbitMQVhost`